packages/utils/src/validate-turnstile-token.ts

```diff
@@ -0,0 +1,31 @@
+const SECRET_KEY = process.env.TURNSTILE_SECRET_KEY;
```

Comment:
Move this file's code to `apps/web/app/api/cloudflare` route handler as a private function.

Reply:
Moved the code